### DISCUSSION TITLE
Unable to bind a user relationship in automation

### DISCUSSION THREAD

**Discussion by mctlisboa** [Budibase/budibase#9903] [Help] [answered]
I have an automation that query all the users table rows and pass it to a row creation automation in another table.

In the new table there is a relationship column with the users table. I am trying to pass the {{ loop.currentItem._id }} binding, but I always get "Error testing automation". But if I remove it or pass the user row id manually the automation runs properly.

I am doing something wrong or its a bug?

<img width="489" alt="Captura de ecrã 2023-03-07, às 06 38 35" src="https://user-images.githubusercontent.com/54587190/223330354-9454b0d7-233d-44b3-9e47-a92d62a15f66.png">

**Comment by andyburgessmd** [comment]
This is similar to my issue I think - [https://github.com/Budibase/budibase/discussions/9894](https://github.com/Budibase/budibase/discussions/9894)
No answer on there yet though.

**Comment by melohagan** [comment]
Hey @mctlisboa 

Could you share the schema of the Benchmark table please and share screenshots of the previous steps in your automation as well.

**Reply by melohagan** [reply]
Ah just noticed you did share those on Discord: https://ptb.discord.com/channels/733030666647765003/1082539077888393307

I'll take a look now

**Comment by mctlisboa** [comment, marked as answer]
I was able to figure it out.

Probably the previous step is returning the user id row as an object like "ro_ta_users_us_76746f125e3d4232bf2515f42d02b89b"

But Budibase automation for relationships probably requires a string. So by wrapping inside a ```{{ JSONstringify trigger.row.Employee }}``` I was able to run the automation successfully.

**Reply by melohagan** [reply]
Nice, great job figuring that one out! I also have a PR out to effectively do that for the dev user in future: https://github.com/Budibase/budibase/pull/9924
